lib/libutee/include/tee_api_defines.h

```diff
@@ -465,8 +470,12 @@
 #define TEE_PARAM_TYPE_SET(t, i) (((uint32_t)(t) & 0xF) << ((i)*4))
```

Comment:
This macro is likely to be used in closed source code we will never review.
And once tee/bm is merged, anyone can/may/will use it. especially regarding traversal timing info support.

There is a way I think to let this legacy macro unchanged.
In `TEE_OpenTASession()` and `TEE_InvokeTACommand()`, where you allocate the bm buffer, you should also force `paramTypes` value to integrate this extra param info, whatever it's already set or not.